packages/mdc-datatable/_mixins.scss

```diff
@@ -0,0 +1,226 @@
+    @include mdc-datatable-divider-color($mdc-datatable-divider-color, $query: $query);
+    @include mdc-datatable-divider-size($mdc-datatable-divider-size, $query: $query);
+    @include mdc-datatable-row-hover-fill-color($mdc-datatable-row-hover-fill-color, $query: $query);
+    @include mdc-datatable-header-row-text-color($mdc-datatable-header-row-text-color, $query: $query);
+    @include mdc-datatable-row-text-color($mdc-datatable-row-text-color, $query: $query);
+    @include mdc-datatable-shape-radius($mdc-datatable-shape-radius, $query: $query);
+    @include mdc-datatable-stroke-size($mdc-datatable-stroke-size, $query: $query);
+    @include mdc-datatable-stroke-color($mdc-datatable-stroke-color, $query: $query);
+    @include mdc-datatable-header-row-height($mdc-datatable-header-row-height, $query: $query);
+    @include mdc-datatable-row-height($mdc-datatable-row-height, $query: $query);
+    @include mdc-datatable-cell-padding(
+      $leading-padding: $mdc-datatable-cell-leading-padding,
+      $trailing-padding: $mdc-datatable-cell-trailing-padding,
+      $query: $query
+    );
+
+    overflow: hidden; // For border-radius to work
+  }
+
+  .mdc-datatable__table {
+    border: 0;
+    border-collapse: collapse;
+  }
+
+  .mdc-datatable__cell {
+    @include mdc-datatable-cell-base-styles_($query: $query);
+  }
+
+  .mdc-datatable__cell-numeric {
+    @include mdc-datatable-cell-base-styles_($query: $query);
+
+    text-align: right;
+  }
+
+  .mdc-datatable__header-cell {
+    @include mdc-typography(subtitle2, $query: $query);
+
+    @include mdc-feature-targets($feat-structure) {
+      text-align: left;
+    }
+  }
+}
+
+@mixin mdc-datatable-fill-color($color, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  @include mdc-feature-targets($feat-color) {
+    @include mdc-theme-prop("background-color", $color);
+  }
+}
+
+@mixin mdc-datatable-header-row-fill-color($color, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  .mdc-datatable__header-row {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-theme-prop("background-color", $color);
+    }
+  }
+}
+
+@mixin mdc-datatable-row-fill-color($color, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  .mdc-datatable__row {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-theme-prop("background-color", $color);
+    }
+  }
+}
+
+@mixin mdc-datatable-selected-row-fill-color($color, $query: mdc-feature-all()) {
+  $feat-color: mdc-feature-create-target($query, color);
+
+  .mdc-datatable__row--selected {
+    @include mdc-feature-targets($feat-color) {
+      @include mdc-theme-prop("background-color", $color);
+    }
+  }
+}
+
+@mixin mdc-datatable-checked-icon-color($color, $query: mdc-feature-all()) {
+  .mdc-datatable__checkbox {
```

Comment:
instead of creating this selector, could we instead just do `.mdc-datatable .mdc-checkbox`?

Reply:
I used this class name to be explicit about checkbox selector that toggles the row selection. That means, the table row will be selected only when clicked on checkbox that has `mdc-datatable__row-checkbox` class names. Other checkbox actions will be ignored.

Renamed from `mdc-datatable__checkbox` => ``mdc-datatable__row-checkbox` to match with design anatomy.